tsconfig.app.json

```
@@ -29,5 +31,5 @@
       "@/*": ["src/*"]
     }
   },
-  "include": ["src"]
+  "include": ["src", "src/types"]
```

```suggestion
  "include": ["src"]
```

이렇게만 있어도 src 하위의 모든 파일들에 대한 tsconfig를 설정한다는 뜻이기 때문에 src/types는 없어도 됩니다.

https://www.typescriptlang.org/tsconfig/#include를 참고해보세요!